diff --git a/es/tree/base-node.js b/es/tree/base-node.js
index a9353c53cd9c1996f811dc6a78a353d7628e7db4..e36033ecacca6fb99c39911201d78ea4c9bae1d0 100644
--- a/es/tree/base-node.js
+++ b/es/tree/base-node.js
@@ -203,11 +203,11 @@ const _sfc_main = defineComponent({
         if (!draggable.value)
           return;
         e.stopPropagation();
-        setDragStatus("dragStart", e);
         try {
           (_a = e.dataTransfer) == null ? void 0 : _a.setData("text/plain", "");
         } catch (error) {
         }
+        setDragStatus("dragStart", e);
       },
       onDragEnd(e) {
         if (!draggable.value)
